library_api/repositories/book_repository.go

```diff
@@ -0,0 +1,28 @@
+package repositories
+
+import (
+	"library_api/entities"
+)
+
+type BookRepository interface {
+	Create(books []entities.Book) (int, error)
+    List() ([]entities.Book, error)
+}
+
+type bookRepository struct {
+	books []entities.Book
+}
+
+func NewBookRepository() BookRepository {
```

Comment:
here you will provide your initial books and set is as a parameter before you provide repo to controllers level